docs/source/reference/mimetype.md

```diff
@@ -22,6 +22,9 @@ As types may contain vendor specific items, a
 - `application/vnd.jupyter.cells`
 - `application/vnd.jupyter.dragindex` used by nbdime
 - `application/x-ipynb+json` for notebooks
+- `text/html` has a metadata option, `'--isolated'` which can annotate an IPython display magic, `%%html`,
```

Comment:
Since this is about mimetype metadata and should be language agnostic, I'd declare the metadata payload first and reference the IPython implementation as an anecdote.

Reply:
Hit merge first. I'll swap the order now. ;-)

Reply:
Works for me!